src/store/Store.js

```diff
@@ -68,6 +78,22 @@ class Store {
   }
 
   /**
+   * Replaces the state for only the keys in the updated state. Notifies all listeners of state change.
+   * @param {object} state the new (partial) redux state
+   */
+  patchState({updated, removed}) {
+    const state = Object.assign({}, this.state, updated);
+
+    removed.forEach(key => {
+      Reflect.deleteProperty(state, key);
```

Comment:
@tshaddix do we have `Reflect.deleteProperty` and `Object.assign` shimmed at build-time to support older browsers?